### PR TITLE
[MMC-1719] Removed automatic snake-casing (Merge Fields in Audience SDK are an u…

### PR DESCRIPTION
**Pull request by michael-patzer**
…nfortunate exception to this rule). Made APIErrorResponse and APIError public. Updated auth header username to 'apikey'.

### CODE REVIEW

File: Anzee/APIErrorResponse.swift [hunk 1]
```diff
@@ -9,8 +9,8 @@
 import Foundation
 
 /// Basic struct to parse any possible API error responses
-struct APIErrorResponse: Codable {
-    let status: Int
-    let type: String
-    let detail: String
+public struct APIErrorResponse: Codable {
+    public let status: Int
```
Comment (shawn-veader) [inline Anzee/APIErrorResponse.swift:13]: If the top level structure is `public` do all the child elements need to be?

Comment (protocolharbor) [reply Anzee/APIErrorResponse.swift:13]: Apparently not. Fixed. :)

**Comment by protocolharbor** [comment]
Made changes here to line things up on the other side: https://github.com/mailchimp/Mailchimp-SDK-iOS/pull/23

**Review by shawn-veader — APPROVED** [review]
👍  Looks good to me. Thanks for the updates.

**Comment by protocolharbor** [comment]
Since it's your personal repo, you'll have to merge.

File: Anzee/APIErrorResponse.swift (hunk 1 above)
Comment (protocolharbor) [reply Anzee/APIErrorResponse.swift:13]: Ok, so Jennifer reminded me the reason these need to be `public` is if they are accessed outside the SDK.

**Comment by protocolharbor** [comment]
@veader Looks like this merged some CLA bot things (last 2 commits) from the MC repo. Probably no harm, but also no value here.

**Pull request merged**